.env

```diff
@@ -61,7 +61,3 @@ SMTP_PASSWORD="f00d"
 # NEW_RELIC_AGENT_ENABLED=true
 # NEW_RELIC_APP_NAME="Open Food Network"
 # NEW_RELIC_LICENSE_KEY="xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
-
-# VINE API settings
-# VINE_API_URL="https://vine-staging.openfoodnetwork.org.au/api/v1"
-# VINE_API_SECRET="XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
```

Comment:
I was a bit too quick here, we stil need the `VINE_API_URL` :sweat_smile:  , I'll fix that this afternoon